aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthWithOTPOptions.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.cognito.options
+
+import com.amplifyframework.auth.options.AuthOTPOptions
+
+/**
+ * Cognito extension of passwordless OTP options to add the platform specific fields.
+ */
+data class AWSCognitoAuthWithOTPOptions
+/**
+ * Advanced options for signing up using Passwordless.
+ * @param clientMetadata Additional custom attributes to be sent to the service such as information about the client
```

Comment:
i think this is optional too right?

Reply:
yeah. I can just remove the word "optional" to avoid confusion.